client/ServerRunner.h

```diff
@@ -9,12 +9,15 @@
  */
 #pragma once
 
+#include "../lib/StartInfo.h"
+
 class CVCMIServer;
+struct StartInfo;
```

Comment:
I think you need namespace wrapper here, not include

```suggestion
VCMI_LIB_NAMESPACE_BEGIN
struct StartInfo;
VCMI_LIB_NAMESPACE_END
```

Reply:
KK, got it already. Unusual it was missing froma  file.